src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,113 @@
+
+    private static final CANSparkMax
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final TalonSRX ELEVATOR_MAG_ENCODER = new TalonSRX(ELEVATOR_MAG_ENCODER_ID);
+    private static final CANcoder ANGLE_CANCODER = new CANcoder(ANGLE_CANCODER_ID);
+
+    static {
+        motors_angle_config();
+        motors_elevator_config();
+        configureElevatorEncoder();
+    }
+
+    private static void motors_elevator_config() {
+        MASTER_ELEVATOR_MOTOR.restoreFactoryDefaults();
+        FOLLOWER_ELEVATOR_MOTOR.restoreFactoryDefaults();
+
+        MASTER_ELEVATOR_MOTOR.setInverted(MASTER_ELEVATOR_INVERTED_VALUE);
+        FOLLOWER_ELEVATOR_MOTOR.setInverted(FOLLOWER_ELEVATOR_INVERTED_VALUE);
+
+        MASTER_ELEVATOR_MOTOR.setIdleMode(ANGLE_NEUTRAL_MODE_VALUE);
+        FOLLOWER_ELEVATOR_MOTOR.setIdleMode(ANGLE_NEUTRAL_MODE_VALUE);
+
+        ELEVATOR_MAG_ENCODER.setSelectedSensorPosition(ELEVATOR_MAG_ENCODER.getSelectedSensorPosition() - ELEVATOR_MAG_ENCODER_OFFSET);
+        ELEVATOR_MAG_ENCODER.setSensorPhase(ELEVATOR_MAG_ENCODER_PHASE_VALUE);
+
+    }
+
+    private static void configureElevatorEncoder() {
+        ELEVATOR_MAG_ENCODER.setSelectedSensorPosition(ELEVATOR_MAG_ENCODER.getSelectedSensorPosition() - ELEVATOR_MAG_ENCODER_OFFSET);
+        ELEVATOR_MAG_ENCODER.setSensorPhase(ELEVATOR_MAG_ENCODER_PHASE_VALUE);
+    }
+
+
+    private static void motors_angle_config() {
+        MASTER_ANGLE_MOTOR.restoreFactoryDefaults();
+        FOLLOWER_ANGLE_MOTOR.restoreFactoryDefaults();
+
+        MASTER_ANGLE_MOTOR.setInverted(MASTER_ANGLE_INVERTED_VALUE);
+        FOLLOWER_ANGLE_MOTOR.setInverted(FOLLOWER_ANGLE_INVERTED_VALUE);
+
+        MASTER_ANGLE_MOTOR.setIdleMode(ANGLE_NEUTRAL_MODE_VALUE);
+        FOLLOWER_ANGLE_MOTOR.setIdleMode(ANGLE_NEUTRAL_MODE_VALUE);
+
+        CANcoderConfiguration configureAngleEncoder = new CANcoderConfiguration();
+        configureAngleEncoder.MagnetSensor.MagnetOffset = ANGLE_CANCODER_OFFSET;
+        configureAngleEncoder.MagnetSensor.AbsoluteSensorRange = ANGLE_ENCODER_SENSOR_RANGE;
+        configureAngleEncoder.MagnetSensor.SensorDirection = ANGLE_CANCODER_SENSOR_DIRECTION;
+        ANGLE_CANCODER.getConfigurator().apply(configureAngleEncoder);
+    }
+
+
+    public enum ArmStates {
+        FIRST_STATE(new Rotation2d(30), 1),
+        SECOND_STATE(new Rotation2d(50), 2),
+        THIRD_STATE(new Rotation2d(100), 3);
+        final Rotation2d angle;
+
+        final double elevatorPosition;
```

Comment:
Missing and extra newline
```suggestion
        THIRD_STATE(new Rotation2d(100), 3);
        
        final Rotation2d angle;
        final double elevatorPosition;
```